plotter/step.go

```diff
@@ -0,0 +1,168 @@
+// Copyright ©2018 The gonum Authors. All rights reserved.
```

Comment:
s/gonum/Gonum/

Reply:
Shouldn't it be changed in some bulk way for the entire repository? Currently non-capitalized version is used in every other file.

Reply:
Yes, after I saw this here I sent #485.

Reply:
OK